Backend/services/authentication/sign_up_service.py

```diff
@@ -1,18 +1,39 @@
-def sign_up(email="", phoneNum="", username="", password="", name="", profilePic="default_pic"):
-    if email != "" and phoneNum == "" and username == "":
-        # make db call
-        pass
+class SignUpService:
+    def __init__(self, body):
+        self.email = body['email']
+        self.phoneNum = body['phoneNum']
+        self.username = body['username']
+        self.password = body['password']
+        self.salt = body['salt']
+        self.name = body['name']
+        self.profilePic = body['profilePic']
 
-    if email == "" and phoneNum != "" and username == "":
-        # make db call
-        pass
+    def sign_up(self):
+        if self.email != '' and self.phoneNum == '' and self.username == '':
+            # make db call and do if else
+            return ['checking database for email', 200]
 
-    if email == "" and phoneNum == "" and username != "":
-        # make db call
-        pass
+        if self.email == '' and self.phoneNum != '' and self.username == '':
+            # make db call
+            return ['checking database for phoneNum', 200]
 
-    if email != "" and phoneNum != "" and username != "" and password != "" and name != "":
-        # make db call
-        pass
+        if self.email == '' and self.phoneNum == '' and self.username != '':
+            # make db call
+            return ['checking database for username', 200]

```

Comment:
use the `flask.make_response`
```suggestion
            return make_response('checking database for username', 200)
```